cfgov/legacy/views/complaint.py

```diff
@@ -45,18 +38,20 @@ def get_context_data(self, **kwargs):
 
         return context
 
-    def get_ccdb_status_json(self, complaint_source):
+    def get_ccdb_status_json(self):
         """Retrieve JSON describing the CCDB's status from a given URL."""
         try:
-            headers = {"accept": "application/json"}
-            res_json = requests.get(complaint_source, headers=headers).json()
-
-        except requests.exceptions.RequestException:
-            logger.exception("CCDB status data fetch failed.")
-            res_json = {}
+            args = {'field': 'all', 'size': '1', 'no_aggs': 'true'}
+            factory = APIRequestFactory()
+            request = factory.get('/search/', args, format='json')
+            response = views.search(request)
+            res_json = response.data
         except ValueError:
             logger.exception("CCDB status data not valid JSON.")
```

Comment:
Because of the shift to calling the view directly, the code no longer deals with JSON at all, as the responses (`response.data`) is just a Python dict. I'm not sure if this `ValueError` clause would ever get called anymore since this change removes the need to call `.json()` on the response.